tool/tctl/common/acl_command.go

```diff
@@ -0,0 +1,242 @@
+/*
+Copyright 2023 Gravitational, Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package common
+
+import (
+	"context"
+	"fmt"
+	"os"
+	"strings"
+	"time"
+
+	"github.com/alecthomas/kingpin/v2"
+	"github.com/gravitational/trace"
+
+	"github.com/gravitational/teleport"
+	"github.com/gravitational/teleport/api/types/accesslist"
+	"github.com/gravitational/teleport/lib/asciitable"
+	"github.com/gravitational/teleport/lib/auth"
+	"github.com/gravitational/teleport/lib/service/servicecfg"
+	"github.com/gravitational/teleport/lib/utils"
+)
+
+// ACLCommand implements the `tctl acl` family of commands.
```

Comment:
Do you think `tctl acl` makes sense for a feature called _Access Lists_?

I wonder if `tctl access` makes more sense? We don't seem to refer to these as ACLs in the code or docs so far.

Reply:
This is from the RFD. Though truthfully I share your skepticism here. I'm not sure what the right call for this is.

Reply:
Could always change this here and go back and update the RFD right? As it's such a small change?

Reply:
I think I'd call in @klizhentas to get his opinion here. This particular bit was subject to a lot of scrutiny.

Reply:
`tctl acl` is fine IMO. ACL literally stands for "access control lists" so I personally don't see an issue.